src/main/java/subway/member/domain/LoginMember.java

```diff
@@ -1,10 +1,23 @@
 package subway.member.domain;
 
+import java.util.Optional;
+
 public class LoginMember {
+
+    private static final Long ID_DEFAULT = 0L;
+    private static final String EMAIL_DEFAULT = "email";
+    private static final int AGE_DEFAULT = 0;
+
     private Long id;
     private String email;
     private Integer age;
 
+    private static LoginMember defaultLoginMember;
+
+    static {
+        defaultLoginMember = new LoginMember(ID_DEFAULT, EMAIL_DEFAULT, AGE_DEFAULT);
+    }
```

Comment:
static 구문을 따로 빼실 필요는 없어요. 멤버변수에서 바로 초기화 하시면 됩니다. 
```suggestion
private static LoginMember defaultLoginMember = new LoginMember(ID_DEFAULT, EMAIL_DEFAULT, AGE_DEFAULT);

Reply:
넵 알겠습니다!